src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -283,32 +290,32 @@ EEG, head localization coils, and anatomical landmarks.
-| EEGCoordinateSystemDescription | OPTIONAL              | [string][]    | Freeform text description or link to document describing the EEG coordinate system system in detail.                                                                                                                                                   |
+| **Key name**                   | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                                                                                                                                                                                                                                 |
+| ------------------------------ | --------------------- | ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| MEGCoordinateSystem            | REQUIRED              | [string][]    | Defines the coordinate system for the MEG sensors. See [Appendix VIII](../99-appendices/08-coordinate-systems.md) for a list of restricted keywords for coordinate systems. If `Other`, provide definition of the coordinate system in `MEGCoordinateSystemDescription`.                                                                                                                                        |
+| MEGCoordinateUnits             | REQUIRED              | [string][]    | Units of the coordinates of `MEGCoordinateSystem`. MUST be `m`, `cm`, or `mm`.                                                                                                                                                                                                                                                                                                                                  |
+| MEGCoordinateSystemDescription | OPTIONAL              | [string][]    | Free-form text description of the coordinate system. May also include a link to a documentation page or paper describing the system in greater detail. REQUIRED if `MEGCoordinateSystem` is `Other`.                                                                                                                                                                                                            |
```

Comment:
See https://bids-specification.readthedocs.io/en/latest/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#timing-parameters.

```suggestion
| MEGCoordinateSystemDescription | OPTIONAL, but REQUIRED if `MEGCoordinateSystem` is `Other` | [string][]    | Free-form text description of the coordinate system. May also include a link to a documentation page or paper describing the system in greater detail.                                                                                                                                                                                                             |
```

Did not attempt to clean up the table spacing.